ghedesigner/manager.py

```diff
@@ -882,6 +882,10 @@ def run_manager_from_cli(input_path, output_directory, validate):
             print("Schema validation error. See previous error message for details.", file=stderr)
             return 1
 
+    if output_directory is None:
+        print('Output directory path must be passed as an argument.')
+        return 1
```

Comment:
If a missing `output_directory` will cause a `return 1`, should it be required in the click argument spec above?

Reply:
Yeah, I would have missed this if we didn't have tests. The CLI has a `validate` option to validate input files and exit. In that case, we don't need the output directory. But if we intend to run, we do need the output dir.

Reply:
Makes perfect sense, thanks!